File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    list_a = []
    for i in range(len(commands)):
        list_a = array[commands[i][0]-1: commands[i][1]]
        list_a.sort()
        answer.append(list_a[commands[i][2]-1])
        
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    for(let i= 0; i < commands.length; i++){
        var temp_array = [];
        temp_array = array.slice(commands[i][0]-1,commands[i][1]);
        temp_array.sort((a, b) => a - b);
        answer.push(temp_array[commands[i][2]-1])
    }
    return answer;
}